mix.exs

```diff
@@ -1,16 +1,18 @@
 defmodule ExSecp256k1.MixProject do
   use Mix.Project
 
+  @source_url "https://github.com/ayrat555/ex_secp256k1"
```

Comment:
@kianmeng can you please set it to `https://github.com/omgnetwork/ex_secp256k1`?

Reply:
> @kianmeng can you please set it to `https://github.com/omgnetwork/ex_secp256k1`?

@ayrat555 Done.